apps/force/src/App.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <Molgenis id="__top" v-model="session">
+    <router-view :session="session" :page="page"></router-view>
+  </Molgenis>
+</template>
+
+<script setup>
```

Comment:
It would be nice if you'd add typescript when creating new vue files ;)

Reply:
It was something we agreed upon in a EMX2 dev meet a while a go that we do that default when creating new components.